README.md

```diff
@@ -31,5 +31,5 @@ Looking for first contributions? Just fork this repository add your name to this
 - [MG Raju](https://github.com/Radioactive92177)
 - [Manas Srivastava](https://github.com/manas1072)
 - [Yusuf Kehinde Hussein](https://github.com/CodingMage)
-- [Aryan Kashyap](https://github.com/iaryankashyap)
```

Comment:
Jatin, you deleted an entry to enter yours. you can't do that. Please fix this error so I can approve this.